tssc/step_implementers/deploy/argocd.py

```diff
@@ -254,7 +254,6 @@ def _run_step(self):
             results = {}
 
             try:
-                git_url = self.get_config_value('helm-config-repo')
```

Comment:
Is there a reason that `self.get_config_value('helm-config-repo')` and `self.get_repo_branch()` are defined/called so differently throughout this file?
I would prefer these to be defined as variables prior to [Line 256](https://github.com/rhtconsulting/tssc-python-package/blob/569e1e83320a758e7566c05271a857267463c3fd/tssc/step_implementers/deploy/argocd.py#L256)

Reply:
updated